processors/ratelimiter/config.go

```diff
@@ -18,8 +18,6 @@ type Config struct {
 	ServicePort uint16 `mapstructure:"service_port"`
 	//Domain  rate limit configuration domain to query. Default collector
 	Domain string `mapstructure:"domain"`
-	// DomainSoftRateLimitThreshold represents the threshold where soft limit window started.
```

Comment:
I'm hoping there are no backwards compatibility issues with removing this config @pavan-traceable ?

Reply:
No backward compatability issues with this.